### PR TITLE
Inklink blog

### PR DESCRIPTION
**Pull request by seanmtracey**

### CODE REVIEW

File: _posts/2017-06-06-inklink.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "InkLink"
+date:   2017-06-06 09:30:30 +0100
+categories: jekyll
```
Comment (Lily2point0) [inline _posts/2017-06-06-inklink.html]: Do we have other categories than jekyll?

File: assets/main.scss [hunk 2]
```diff
@@ -228,6 +228,8 @@ main article img{
   display: flex;
   border: 5px solid white;
   box-shadow: 0 1px 3px black;
+  background: white;
```
Comment (Lily2point0) [inline assets/main.scss:231]: Do we want to force white background, or should it be baked into the png?

File: _posts/2017-06-06-inklink.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+<o>InkLink can recognise a variety of images as long as they’re the focal point of the capture. It could even recognise the daily crossword grid (blank or partially filled in).</p>
+
+<p>To achieve this result, we explored number of third-party services that focused on AR.</p>
+
+<h2>Technical details</h2>
+<p>After looking into Image Retrieval (the technique employed by Google reverse image search), it became apparent that we didn’t need to identify elements within an image, just recognise it. That’s why we simplified our approach and are now focusing on Image Recognition. To get it working, we compared 3 different services: Blippar, Aurasma, and CraftAR by Catchoom.</p>
```
Comment (Lily2point0) [inline _posts/2017-06-06-inklink.html]: Missing links to the 3 services we tried:
[Blippar](https://blippar.com/en/)
[Aurasma](https://www.aurasma.com/)
[Catchoom](https://catchoom.com/solutions/image-recognition/)

Comment (seanmtracey) [reply _posts/2017-06-06-inklink.html]: 👍 Will rectify

File: assets/main.scss (hunk 2 above)
Comment (seanmtracey) [reply assets/main.scss:231]: I can't see any harm in forcing it. I don't think images should be responsible for their own presentation in a page.

File: _posts/2017-06-06-inklink.html (hunk 1 above)
Comment (seanmtracey) [reply _posts/2017-06-06-inklink.html]: The categories are arbitrary. This is a hangover from the file that I copied to base this post on. The only place it appears at current is on the home page card. I'll change it to "experiment".

**Review by Lily2point0 — APPROVED** [review]
Did a small markup fix (missing <p> tags). Rest looks good.

**Comment by railsagainstignorance** [comment]
add a mention that it also works off projected screens, as an alternative to QR codes at presentations
.

**Pull request merged**